### PR TITLE
Adding T0_CH_CSCS_HPC to overflow-sites

### PR DESCRIPTION
**Pull request by SvenjaPflitsch**

### CODE REVIEW

**Comment by SvenjaPflitsch** [comment]
The @mcremone @thongonary @vlimant  I have added the mapping for CSCS. There were some mappings that were done for T0 but not HLT, which I have listed below. Pleas let me know if they should also be added. 

Line 99-100 of org-file: 
---------------------
## temptatively
mapping['T0_CH_CERN'].append( 'T2_CH_CERN' )

Line 118 of org file: 
--------------------
mapping['T0_CH_CERN'].append( 'T1_US_FNAL' )

**Comment by thongonary** [comment]
Thanks Svenja. Everything looks fine to me. I don't think having CSCS overflows to T2 CERN and FNAL is necessary, but I could be wrong. If there is no objection from Jean-Roch and Matteo by the end of today, I will merge this PR.

**Comment by vlimant** [comment]
https://cms-gwmsmon.cern.ch/prodview/T2_CH_CSCS_HPC is already in production
https://cms-gwmsmon.cern.ch/prodview/T0_CH_CSCS_HPC on the other hand is the site that needs to be used asap.

**Comment by thongonary** [comment]
Oh thanks for spotting that! @SvenjaPflitsch can you please change T2 to T0?

**Comment by SvenjaPflitsch** [comment]
Sure I will do it right away.

**Comment by mcremone** [comment]
And please no overflow from CSCS to T2 CERN and FNAL

File: Unified/equalizor.py [hunk 1]
```diff
@@ -40,6 +40,10 @@ def equalizor(url , specific = None, options=None):
     if options.hlt: use_HLT = True
     if use_HLT: over_rides.append('T2_CH_CERN_HLT')
     
+    use_CSCS = ('T2_CH_CSCS_HPC' in UC.get("site_for_overflow"))
+    if options.cscs: use_CSCS = True
+    if use_CSCS: over_rides.append('T2_CH_CSCS_HPC')
```
Comment (thongonary) [inline Unified/equalizor.py]: This also needs to be changed

File: Unified/equalizor.py [hunk 2]
```diff
@@ -40,6 +40,10 @@ def equalizor(url , specific = None, options=None):
     if options.hlt: use_HLT = True
     if use_HLT: over_rides.append('T2_CH_CERN_HLT')
     
+    use_CSCS = ('T2_CH_CSCS_HPC' in UC.get("site_for_overflow"))
```
Comment (thongonary) [inline Unified/equalizor.py]: and here of course

**Pull request merged**